src/ununpack/agent/standalone.c

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (C) 2020 Siemens AG
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+version 2 as published by the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+/*
+**
+ * \file
+ * \this is provide a dummy values for library function to make ununpack agent standalone
```

Comment:
Please remove the `\` from the beginning as `\this` is not a valid doxygen command.
```suggestion
 * This is provide a dummy values for library function to make ununpack agent standalone
```

Reply:
done

Reply:
done